decko/README.rdoc

```diff
@@ -81,7 +81,23 @@ Run the following:
 
 ====4. Restart your server.
 
-===Upgrading pre-gem Wagn sites
+===Upgrade to Decko from Wagn
```

Comment:
just realized we didn't have this anywhere.

Reply:
technically we could automate some of this, but running `decko update` would break unless most of it was already done...